### PR TITLE
Setup and mobile first PR

### PR DESCRIPTION
**Pull request by MudasirSherwani**
### Setup and Mobile First Branch

1. Install Linters
2. Add Index.html file
  - Header with nav bar
  - headline section
3. Add styles.css file
4. Add Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
...
+
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
```
Comment (Ismailco) [inline README.md]: - [ ] Kindly remove unused headers in your README, so it looks more professional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      <div class="developer-title align-center" >
+        <h1 class="developer-name">Mr. Mudasir</h1>
+        <h2 class="software-developer">Software Developer</h2>
+      </div>
+        <p class="banner-text-pera">I can help you build a product, feature or website Look through some of my 
+          work and experience! if you like what you see and have a project you need coded, don't hasitate to 
+          contact me.
+        </p>
+    </section>
+</body>
```
Comment (Ismailco) [inline index.html:73]: - [ ] You need to finish the header section by adding the social links and other lists, kindly consider adding this to your page

Screenshot_20220817_203712

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @MudasirSherwani, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — APPROVED** [review]
Hi @MudasirSherwani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy (1)

## Highlights 💯 
- Very well following the suggestions of the previous reviewer.

## Optional suggestions

 [OPTIONAL] I kindly inform you that the previous reviewer overlooked that the white spots of the design should be background images and not floating div, it is highly recommended that you make that change so as not to have problems in future projects that you present, good luck! 🔥 🔥 🔥 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**